molgenis-app/src/main/java/org/molgenis/app/promise/mapper/ParelMapper.java

```diff
@@ -280,14 +290,29 @@ private Entity toAgeType(String ageType)
 		List<String> unknown = Lists.newArrayList();
 		for (Entity sample : promiseSampleEntities)
 		{
-			String type = sample.getString("MATERIAL_TYPES");
-			if (materialTypesMap.containsKey(type))
+			String tissue = sample.getString("MATERIAL_TYPES_SUB");
```

Comment:
I (obviously! :) ) don't know how the mapping works but would expect that the MATERIAL_TYPES_SUB only contains tissue types if the MATERIAL_TYPES attribute contains "Tissue" or "weefsel" or whatever.